package.json

```diff
@@ -3,7 +3,7 @@
   "name": "fra-platform",
   "engines": {
     "node": "20.11.1",
-    "yarn": "^1.22.22"
+    "yarn": "^1.22.19"
```

Comment:
I think the node docker image is using yarn 1.22.19+ so this was required change vs installing yarn 1.22.22

I think it should be ok